Formula/bnfc.rb

```diff
@@ -126,7 +126,7 @@ def install
       space = " "
       check_out_j = <<~EOS
 
-        Parse Succesful!
+        Parse Successful!
```

Comment:
This should not be changed, we're matching the tool's output (typo included)

Reply:
Done, thank you for your review.